web/src/user_groups.ts

```diff
@@ -305,6 +305,19 @@ export function is_setting_group_empty(setting_group: GroupSettingValue): boolea
     return true;
 }
 
+export function is_setting_group_set_to_nobody_group(setting_group: GroupSettingValue): boolean {
```

Comment:
We have a `is_setting_group_empty` function but that returns true even if the setting is set to a user defined group with no members or subgroups. But in that case we can actually add a user to that group, so the content access is not lost entirely.